app/src/util/Wallets2Tokens.js

```diff
@@ -6,7 +6,7 @@ import Web3 from 'web3'
 import EosWrapper from '@/util/EosWrapper'
 // import wallet from '../router/wallet'
 const eos = new EosWrapper()
-
+const eosTestnet = new EosWrapper('http://140.82.56.143:8888')
```

Comment:
Is this correct of should it be `process.env[store.state.settings.network].EOS_HISTORYAPI`

Reply:
This is taken care of in the most recent commit !